connector/src/main/java/org/geysermc/connector/network/translators/java/entity/JavaEntitySetPassengersTranslator.java

```diff
@@ -141,7 +141,17 @@ private void updateOffset(Entity passenger, EntityType mountType, GeyserSession
                 yOffset = 1.3f;
                 break;
             case STRIDER:
-                yOffset = passenger.getEntityType() == EntityType.PLAYER ? 2.8200102f : 1.6f;
+                switch(passenger.getEntityType()) {
+                    case EntityType.SPIDER:
```

Comment:
I assume you meant `EntityType.STRIDER` here.

Reply:
I'm sorry.
I pressed too fast and missed a button.

Reply:
There is absolutely no need to apologize! The point of us reviewing is to catch those small mistakes.